src/Java.Interop.Tools.Maven/Repositories/MavenRepository.cs

```diff
@@ -0,0 +1,38 @@
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Net.Http;
+using Java.Interop.Tools.Maven.Models;
+
+namespace Java.Interop.Tools.Maven.Repositories;
+
+public class MavenRepository : IMavenRepository
+{
+	readonly string base_url;
+	static readonly HttpClient client;
+
+	public string Name { get; }
+
+	public MavenRepository (string baseUrl, string name)
+	{
+		Name = name;
+		base_url = baseUrl.TrimEnd ('/');
+	}
+
+	static MavenRepository ()
```

Comment:
Any reason to have an explicit static constructor vs. just initializing in the field?

Reply:
Fixed.